contrib/raspberrypi3/run_electrumx.sh

```diff
@@ -0,0 +1,38 @@
+###############
+# run_electrumx
```

Comment:
It is not necessary. It is a tip to allow for easier testing and deployment. I'm ok if you would rather delete it.